pkg/abaputils/abaputils.go

```diff
@@ -221,12 +222,23 @@ func ReadAddonDescriptor(FileName string) (AddonDescriptor, error) {
 	if err != nil {
 		return addonDescriptor, errors.New(fmt.Sprintf("Could not parse %v", FileName))
 	}
-
 	err = json.Unmarshal(jsonBytes, &addonDescriptor)
 	if err != nil {
 		return addonDescriptor, errors.New(fmt.Sprintf("Could not unmarshal %v", FileName))
 	}
-
+	if len(addonDescriptor.Repositories) == 0 {
```

Comment:
The purpose of "ReadAddonDescriptor" is to parse the YML file. This YML file does not only contain repositories. It is not responsible to verify the content of the YML file.
Therefore, I would leave out the introduced checks. The caller of this function needs to verify if the returned structure fits the purpose.